packages/js/create-product-editor-block/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog 
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0](https://www.npmjs.com/package/@woocommerce/create-product-editor-block/v/1.0.0) - 2023-06-28 
+
+-   Minor - Initial version of @woocommerce/create-product-editor-block [#38263]
+-   Minor - Do not add script prop to generated block.json [#38496]
+
+[See legacy changelogs for previous versions](https://github.com/woocommerce/woocommerce/blob/68581955106947918d2b17607a01bdfdf22288a9/packages/js/create-product-editor-block/CHANGELOG.md).
```

Comment:
It seems like there is something wrong with this link. It's showing 404 error

![Screenshot 2023-06-28 at 13 31 05](https://github.com/woocommerce/woocommerce/assets/1314156/3539baf8-d6a0-47d6-b0d0-1248e520fc53)